nextcord/channel.py

```diff
@@ -794,6 +816,450 @@ def archived_threads(
         :class:`Thread`
             The archived threads.
         """
+
+        return ArchivedThreadIterator(
+            self.id, self.guild, limit=limit, joined=joined, private=private, before=before
+        )
+
+
+class ForumChannel(abc.GuildChannel, Hashable):
+    """Represents a Discord guild forum channel.
+
+    .. container:: operations
+
+        .. describe:: x == y
+
+            Checks if two channels are equal.
+
+        .. describe:: x != y
+
+            Checks if two channels are not equal.
+
+        .. describe:: hash(x)
+
+            Returns the channel's hash.
+
+        .. describe:: str(x)
+
+            Returns the channel's name.
+
+    Attributes
+    ----------
+    id: :class:`int`
+        The snowflake ID of this channel.
+    guild: :class:`Guild`
+        The guild this channel belongs to.
+    name: :class:`str`
+        The name of this channel.
+    category_id: Optional[:class:`int`]
+        The ID of the :class:`CategoryChannel` this channel belongs to, if any.
+    topic: :class:`str`
+        The topic of this channel, if any.
```

Comment:
> The `topic` field on a forum channel is what is shown in the "Guidelines" section visually
https://discord.com/developers/docs/topics/threads#forums
```suggestion
        The topic of this channel, if any. This is what is shown in the "Guidelines" section visually.
```